exercises/book-store/canonical-data.json

```diff
@@ -0,0 +1,76 @@
+{
+   "#": [
+     "Not sure what to type here, come back to this later."
+   ],
+   "cases": [
+      {
+         "description": "TestSingleBookPrice",
```

Comment:
what is the difference between this and "TestTotalBasketPriceSingleBook"?

If the only function under test in this exercise is "Given this basket, what is its price?" then it seems there is no need to have this one.